sample-app/src/components/cards/StandardCard.tsx

```diff
@@ -1,11 +1,44 @@
+import { CompositionMethod, useComposedCssClasses } from '../../hooks/useComposedCssClasses';
 import { CardProps } from '../../models/cardComponent';
 
 export interface StandardCardConfig {
   showOrdinal?: boolean
 }
 
 export interface StandardCardProps extends CardProps {
-  configuration: StandardCardConfig
+  configuration: StandardCardConfig,
+  customCssClasses?: StandardCardCssClasses,
+  compositionMethod?: CompositionMethod
+}
+
+export interface StandardCardCssClasses {
+  container?: string,
+  header?: string,
+  body?: string,
+  descriptionContainer?: string,
+  ctaContainer?: string,
+  cta1?: string,
+  cta2?: string,
```

Comment:
For my own edification, what guidelines do you use for determining what rises to the level of a block vs an element? Would `ctaContainer__cta1` and `ctaContainer__cta2` have worked here?

Reply:
I'm treating the component as the block, so these represented in BEM would look like: `StandardCard__ctaContainer`, `StandardCard__cta1`, and `StandardCard__cta2`. Therefore everything after the double underscore is an element. And I'm using a triple underscore to indicate modifiers. See the [explanation here](https://github.com/yext/answers-headless-react/pull/67#discussion_r760471334)